components/sidebar/sidebar_category_list/sidebar_category_list.tsx

```diff
@@ -2,32 +2,286 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import Scrollbars from 'react-custom-scrollbars';
```

Comment:
Can we dump `react-custom-scrollbars` in favor of default scrollbars?

Reply:
I'd be in favour of this, as long as it doesn't mess with all the auto scrolling stuff we already do. @hmhealey sound good to you?

Reply:
It doesn't affect the auto scrolling as far as I know. I think it's just for custom styling, so as long as that's not a problem with browsers we support, I'm good with it

Reply:
Okay all done. One thing for you @bradjcoughlin, we'll want to auto-hide the scrollbar when the sidebar isn't hovered/focused, which means we'll want to only apply `overflow-y: scroll` on hover. Currently I have an inline style to make it show up all the time.